azext_edge/tests/edge/orchestration/test_instance_unit.py

```diff
@@ -104,3 +109,58 @@ def test_instance_list(mocked_cmd, resource_group_name: str, records: int):
     instances = Instances(mocked_cmd)
     result = list(instances.list(resource_group_name=resource_group_name))
     assert result == mock_instance_records["value"]
+    assert len(mocked_responses.calls) == 1
+
+
+@pytest.mark.parametrize(
+    "description",
+    [None, generate_random_string()],
+)
+@pytest.mark.parametrize(
+    "tags",
+    [None, {"a": "b", "c": "d"}],
+)
+def test_instance_update(mocked_cmd, mocked_responses: responses, description: Optional[str], tags: Optional[dict]):
```

Comment:
this has a possibility to become much longer (assuming that update will have other parameters later on) - when that time comes, I would like this file to be broken into different files (to avoid long files)

Reply:
We'll see how long things get.